src/main/java/eva/ui/Ui.java

```diff
@@ -0,0 +1,144 @@
+package eva.ui;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import eva.exceptions.TaskException;
+import eva.tasks.Task;
+
+/**
+ * Represents the user interface of the program. Handles user input and prints messages to the user.
+ */
+public class Ui {
+    private final String INTRO = "Hello! I'm Eva. \nWhat can I do for you? \n";
+    private final String END = "Bye. Hope to see you again soon!";
```

Comment:
Perhaps a more descriptive variable name might be better here? In case `END` may cause confusion for some.
```suggestion
    private final String INTRO_MESSAGE = "Hello! I'm Eva. \nWhat can I do for you? \n";
    private final String END_MESSAGE = "Bye. Hope to see you again soon!";
```